docs/DeveloperGuide.md

```diff
@@ -277,6 +277,59 @@ For **Alternative 2**, we found it to be too complex. The user's input has to be
 By converting it to an `Event` class early, we can work at a higher level of abstraction.
 Other methods do not have to worry about string's format, and can focus on handling it as an `Event` class.
 
+### 3.3 Persistent block and room settings
+
+#### 3.3.1 Implementation
+This feature is implemented by making use of a json file to store the blocks and rooms info of the Hall. It does this by defining all available block and rooms in an editable json file. 
```

Comment:
We need to standardize whether or not to capitalize hall. I feel there's no need to capitalize it. We can KIV this for future discussions if needed.